### PR TITLE
twis39

### PR DESCRIPTION
**Pull request by larsbergstrom**
r? @Manishearth

[Review on Reviewable](https://reviewable.io/reviews/servo/blog.servo.org/37)

### CODE REVIEW

File: _posts/2015-10-26-twis-39.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+[Michael Wu](https://github.com/michaelwu/), the originator of our Gonk port (aka, Boot2Servo) and frequent hacker on our SpiderMonkey integration,
+has also been granted reviewer priviledges.
+
+###Notable additions
+
+ - Paul Rouget added a wonderful [quickstart hacking document](https://github.com/servo/servo/pull/8165) to the repo to help with onboarding first-time contributors
+ - Florian Merz [implemented a caret](https://github.com/servo/servo/pull/7761) for input elements
+ - Tetsuharu Ohzeki [added](https://github.com/servo/servo/pull/8183) `webkitMatchesSelector`
+ - Matt Brubeck [enabled multitouch](https://github.com/servo/servo/pull/8163)
+ - Ben Schults [implemented timer ordering guarantees](https://github.com/servo/servo/pull/7450)
```
Comment (jdm) [inline _posts/2015-10-26-twis-39.md]: Schulz

File: _posts/2015-10-26-twis-39.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+We have TWO new reviewers this week! [Eli Friedman](https://github.com/eefriedman) has been doing fantastic work reviewing some DOM/Content-related
+PRs and we're looking forward to burdening them with even more in the future!
+
+[Michael Wu](https://github.com/michaelwu/), the originator of our Gonk port (aka, Boot2Servo) and frequent hacker on our SpiderMonkey integration,
+has also been granted reviewer priviledges.
+
+###Notable additions
+
+ - Paul Rouget added a wonderful [quickstart hacking document](https://github.com/servo/servo/pull/8165) to the repo to help with onboarding first-time contributors
+ - Florian Merz [implemented a caret](https://github.com/servo/servo/pull/7761) for input elements
```
Comment (jdm) [inline _posts/2015-10-26-twis-39.md]: s/input/textarea/

**Comment by jdm** [comment]
Let's mention https://github.com/servo/servo/pull/8008 too.

**Comment by jdm** [comment]
And https://github.com/servo/servo/pull/7951.

**Comment by larsbergstrom** [comment]
@jdm better? :-)

**Comment by jdm** [comment]
Looks good to me!

**Pull request merged**